src/librustc_mir/build/block.rs

```diff
@@ -126,7 +126,7 @@ impl<'a, 'gcx, 'tcx> Builder<'a, 'gcx, 'tcx> {
                             None,
                             remainder_span,
                             lint_level,
-                            &pattern,
+                            &[pattern.clone()],
```

Comment:
is this just to convert from `&T` to `&[T]`? I know there was some helper for doing this *somewhere*...

Reply:
Oh, hey, the libs team finally gave in:

https://doc.rust-lang.org/std/slice/fn.from_ref.html

well, I guess it's still unstable.

Reply:
i was lazy. will fix.